hubconf.py

```diff
@@ -6,7 +6,11 @@
     import torch
     model = torch.hub.load("Megvii-BaseDetection/YOLOX", "yolox_s")
 """
-dependencies = ["torch"]
+from tools.hub import ensure_installed
+
+HUB_REQUIREMENTS = ["loguru", "numpy", "torchvision", "thop", "onnx",
```

Comment:
Did you check them one by one? why `onnx` and `tabulate` is needed here?

Reply:
I've removed `numpy` (installed with `torchvision`) and `onnx`. Not really sure where did `onnx` come from.
I'm getting the `No module named 'tabulate'` error in https://github.com/Megvii-BaseDetection/YOLOX/blob/ce4b996917b6f6983247c90c933759896c04091c/yolox/exp/base_exp.py#L9
This file is used to load models, I guess.